google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -420,14 +419,22 @@ private Builder() {
       // Per-method settings using baseSettings for defaults.
       readRowsSettings = ServerStreamingCallSettings.newBuilder();
       readRowsSettings
-          .setRetryableCodes(baseDefaults.readRowsSettings().getRetryableCodes())
+          .setRetryableCodes(
+              ImmutableSet.<Code>builder()
```

Comment:
Please add a comment to explain why we are diverging from the standard gapic settings:

```
// Allow retrying ABORTED statuses. These will be returned by the server when the client is too slow to read the rows.
// This makes sense for the java client because retries happen after the row merging logic. Which means that the retry
// will not be invoked until the current buffered chunks are consumed. 
```